app/src/main/java/org/mozilla/fenix/tabtray/TabTrayDialogFragment.kt

```diff
@@ -57,6 +57,9 @@ class TabTrayDialogFragment : AppCompatDialogFragment(), TabTrayInteractor {
     override fun onConfigurationChanged(newConfig: Configuration) {
         super.onConfigurationChanged(newConfig)
 
+        val isLandscape = newConfig.orientation == Configuration.ORIENTATION_LANDSCAPE
```

Comment:
could we combine this with the next check?

Reply:
doh, I meant to do that! Good catch!